Documentation/ExtensionArchitecture/Tutorials/Tea/DirectoryStructure.rst

```diff
@@ -0,0 +1,189 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    Tutorial Tea; Directory structure
+..  _extbase_tutorial_tea_extension_configuration:
+
+============================
+Create a directory structure
+============================
+
+Extbase requires or defaults to a certain directory structure. It is considered
+best practise to always stick to this structure.
+
+On the first level EXT:tea has the following structure:
+
+..  code-block:: none
+    :caption: Directory structure of EXT:tea
+
+    $ tree typo3conf/ext/tea
+    ├── Classes
+    ├── Configuration
+    ├── Documentation
+    ├── Resources
+    ├── Tests
+    ├── composer.json
+    ├── ext_emconf.php
+    ├── ...
+    └── README.md
+
+Directory :file:`Classes`
+-------------------------
+
+The folder :file:`Classes` has to contain all PHP classes provided by the
+extension. Otherwise they are not available in the
+:ref:`autoloading <autoload>`.
+
+See also the general chapter on the folder :ref:`extension-classes`.
+
+In the :file:`composer.json` we defined that all PHP classes should
+automatically be loaded from the directory :file:`Classes`:
+
+..  tabs::
+
+    ..  group-tab:: Composer
+
+        ..  code-block:: json
+            :caption: EXT:tea/composer.json, extract
+            :emphasize-lines: 4
+
+            {
+                "autoload": {
+                    "psr-4": {
+                        "TTN\\Tea\\": "Classes/"
+                    }
+                }
+            }
+
+    ..  group-tab:: Legacy
```

Comment:
This example does not match with the sentence above ("In the :file:`composer.json` we defined") - why then ext_emconf.php?

Reply:
Fixed